settings.py

```diff
@@ -2077,3 +2077,20 @@ def _pkce_required(client_id):
 # Disable Datadog trace startup logs by default
 # https://docs.datadoghq.com/tracing/troubleshooting/tracer_startup_logs/
 os.environ['DD_TRACE_STARTUP_LOGS'] = os.environ.get('DD_TRACE_STARTUP_LOGS', 'False')
+
+# These are config settings for the google oauth handshake to get a user token\
+# This is the Google Cloud Platform secret settings config file
+GOOGLE_OATH_CONFIG = {"web":
+        {"client_id": "699334824903-fsomonub18fa0en5c7t0ao01l0fhduil.apps.googleusercontent.com",
```

Comment:
this should not be publicly viewable in `settings.py` I recommend an empty dict and then overwriting it with localsettings in commcare-cloud. Also, please change all the client secrets and ids!